mlrun/lists.py

```diff
@@ -199,7 +199,7 @@ def to_df(self, flat=False):
         """convert the artifact list to a dataframe"""
         rows = self.to_rows()
         df = pd.DataFrame(rows[1:], columns=rows[0])
-        df["updated"] = pd.to_datetime(df["updated"])
+        df["updated"] = pd.to_datetime(df["updated"]).dt.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
this method is used in many places and may have side effects we don't want
better resolve it specifically for the cli here https://github.com/mlrun/mlrun/blob/development/mlrun/__main__.py#L760
see lines 760 - 770

Reply:
fixed